pass_import.py

```diff
@@ -255,19 +280,21 @@ def _clean_protocol(self, string):
 
     def _clean_group(self, string):
         """Remove invalids caracters in a group. Convert separator to os.sep."""
-        caracters = dict(zip(self.invalids, [self.separator]*len(self.invalids)))
+        caracters = dict(
+            zip(self.invalids, [self.separator]*len(self.invalids)))
         caracters['/'] = os.sep
         caracters['\\'] = os.sep
         return self._replaces(caracters, string)
 
     def _convert(self, string):
         """Convert invalid caracters by the separator in a string."""
-        caracters = dict(zip(self.invalids, [self.separator]*len(self.invalids)))
+        caracters = dict(
+            zip(self.invalids, [self.separator]*len(self.invalids)))
         return self._replaces(caracters, string)
 
     def _clean_cmdline(self, string):
         """Make the string more command line friendly."""
-        return self._replaces(self.cleans, string)
+        return self._replaces(self.cleans, string.strip())
```

Comment:
Not in the scope. Furthermore, the purpose of this function, when called is to replace spaces, not to remove them.